README.md

```diff
@@ -32,7 +32,7 @@ Both user and code documentation is available [here](http://umpire.readthedocs.i
 
 The Umpire [tutorial](https://umpire.readthedocs.io/en/develop/tutorial.html) provides a step by step introduction to Umpire features.
 
-If you have build problems, we have comprehensive [build sytem documentation](https://umpire.readthedocs.io/en/develop/advanced_configuration.html) too!
+If you have build problems, we have a comprehensive [build sytem documentation](https://umpire.readthedocs.io/en/develop/advanced_configuration.html) too!
```

Comment:
I don't think this is a grammatical improvement over the old wording...

Reply:
Actually, it looks incorrect after the change.

Reply:
Really? Ok. Reverted https://github.com/LLNL/Umpire/pull/133/commits/73feee02e798bc373895e67b1ed0fae955163c26

Reply:
I have noticed a typo though!

```suggestion
If you have build problems, we have comprehensive [build system documentation](https://umpire.readthedocs.io/en/develop/advanced_configuration.html) too!
```

Reply:
https://github.com/LLNL/Umpire/pull/133/commits/b5dfa95f7767e2ac9e5bf42ba9027b693f55dd7c